app/src/main/java/ch/epfl/qedit/backend/database/MockDBService.java

```diff
@@ -25,12 +24,21 @@
     /** This class simulates a quiz that is stored in Firestore */
     private static class MockQuiz {
 
+        private boolean treasureHunt;
```

Comment:
I did the exact same code in my PR ;). Great minds think alike xD... joking...

Reply:
joking... you say that because i am not a great mind? It is hurtful. I am hurt. <smiley face the is crying>